monitoring/mock_uss/scdsc/muss_reports.py

```diff
@@ -0,0 +1,90 @@
+import datetime, json
+from typing import Dict, List, Optional
+from implicitdict import ImplicitDict
+from monitoring.monitorlib import fetch
+from pathlib import Path
+
+InteractionID = str
+
+
+class MockUssTestContext(ImplicitDict):
+    test_name: str
+    """Name of the test"""
+
+    test_case: str
```

Comment:
It doesn't seem like mock_uss is going to know what test case is currently being run by uss_qualifier; do we need/want this field?  Same with test_name and many of the fields in Issue.  Can we just record the interactions and their message signing statuses?

I would imagine uss_qualifier could add message signing evaluation information into the main report if mock_uss would return report content in its "stop message signing" endpoint or similar.

Reply:
We have given different values for the test_name and test_case for the interactions and issues in mock_uss reports. 
We want to report the issues, just interactions will not help find the issues easily.

Reply:
How will these values be populated though?  mock_uss is a separate executable from uss_qualifier and so therefore does not generally know what uss_qualifier is doing (what test scenario/case/step/etc uss_qualifier is on).

One approach could be to simply collect message signing information in mock_uss after being instructed to start doing so, and then transfer that information back to the client (uss_qualifier) when instructed to stop collecting message signing information.  With that set of information, uss_qualifier could then incorporate that information in its single standard report.